src/main/kotlin/no/nav/familie/ef/sak/service/steg/BehandlingSteg.kt

```diff
@@ -81,8 +76,8 @@ enum class StegType(val rekkefølge: Int,
         return this.name.replace('_', ' ').toLowerCase().capitalize()
     }
 
-    fun kommerEtter(steg: StegType): Boolean {
-        return this.rekkefølge > steg.rekkefølge
+    fun kommerEtter(steg: StegType, behandlingType: BehandlingType): Boolean {
+        return this == steg.hentNesteSteg(behandlingType)
```

Comment:
Litt usikker på denne @henninghaakonsen @ninakjek 
Ønsker man mulighet for å hoppe over steg?